adele-booking/src/main/java/com/microservicesteam/adele/booking/boundary/web/BookingController.java

```diff
@@ -0,0 +1,40 @@
+package com.microservicesteam.adele.booking.boundary.web;
+
+import com.microservicesteam.adele.booking.domain.BookingRequest;
+import com.microservicesteam.adele.booking.domain.BookingResponse;
+import com.microservicesteam.adele.booking.domain.BookingService;
+import com.microservicesteam.adele.ticketmaster.model.Ticket;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import static java.util.Comparator.comparingInt;
+import static java.util.stream.Collectors.toList;
+
+@RestController
+@RequestMapping("/rs/api")
+public class BookingController {
```

Comment:
I am not sure I would like to see /rs in the path on behalf of a user.
This is not a programming interface therefore I would not use /api either.

path's I can think of:
`/events`
`/events/1`
`/events/1/tickets`
`/events/1/tickets/2`
`/events/1/tickets?bookingId=abc-123`

or

`/tickets?eventId=1`
`/tickets?eventId=1&bookingId=abc-123`